packages/hop-node/src/constants/constants.ts

```diff
@@ -24,6 +24,14 @@ export const AvgBlockTimeSeconds = {
   xDai: 5
 }
 
+export const SettlementCostPerTx: Record<string, number> = {
```

Comment:
> `SettlementCostPerTx`

If I understand correctly, the values here are gas limits, so I think a variable name like `SettlementGasLimitPerTx` would be better since Cost implies gasLimit*gasPrice

Reply:
You are correct. Good call, updated with 9991eb7f7ee2efce6b373f6ded989f8779617319